outputs.tf

```diff
@@ -1,3 +1,3 @@
 output "ami_id" {
-  value = "${aws_ami_from_instance.create_ami.id}"
+  value = "${module.tf_label.id}"
```

Comment:
`aws_ami_from_instance.create_ami.id` is authoritative. Use that as the source of truth.